src/ekore/anomalous_dimensions/unpolarized/space_like/as4/__init__.py

```diff
@@ -6,6 +6,7 @@
 import numba as nb
 import numpy as np
 
+from . import fhmruvv_approximations
```

Comment:
```suggestion
from . import fhmruvv
```
I guess the acronym is unique enough